Src/Core/Entity/Button.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+** EPITECH PROJECT, 2022
+** arcAAAAAAAAAAAAAAAAade
+** File description:
+** Button
+*/
+
+#include "Button.hpp"
+
+Button::Button()
+{
+}
+
+Button::~Button()
+{
+}
+
+bool Button::isClicked(int x, int y)
```

Comment:
FYI: int, float etc... functions params to represent a position are to be replaced with a templated vector class